specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -6044,6 +6044,11 @@
     },
     "SemanticSettings": {
       "properties": {
+        "defaultConfiguration": {
```

Comment:
This will need sign off from the API stewardship board as this is considered a breaking change as it's adding a new optional property that didn't exist before.

Reply:
My understanding is that optional, additive changes were okay and do not require a separate sign-off from the API stewardship board. We went by this policy when we last did swagger changes in late 2021: https://github.com/Azure/azure-rest-api-specs/pull/16777. Has this changed since then?

Adding @BevLoh to chime in.

Reply:
We still need to get signoff on the exception from the API board. See the DP swagger process doc for details.

Reply:
CC @mikekistler @JeffreyRichter

Reply:
Yes, **_any_** change to an API contract **_requires_** a new api-version. We used to have an exception for somethings but we no longer do as it was causing too much confusion and people were trying to abuse the system.

Reply:
@JeffreyRichter, this is news to me. ACS has been adding optional properties to existing API versions for quite some time. As additional data point, we did not get any feedback from our customers that this has been causing confusion. Just to extra confirm, the email exception process that we have agreed upon and used in the past is no longer supported. The reason that I'm asking the team will need to figure out how to schedule additional work to add a new api version which long term also carries extra cost maintenance to the code base.

Reply:
Yes, any change to an API contract requires a new api-version - this is just the best thing for customers and customer support services (when working with customers). It simplifies our rules and attempted abuse. I'm sorry this adversely affects ACS but we felt that this was the best choice after weighing all options.